src/ifcblenderexport/blenderbim/export_ifc.py

```diff
@@ -1633,25 +1722,43 @@ def create_cartesian_point(self, x, y, z=None):
     def create_direction(self, vector):
         return self.file.createIfcDirection((vector.x, vector.y, vector.z))
 
+    def relate_voids_elements(self):
+        for relate_object, related_voids in self.ifc_parser.rel_voids_elements.items():
```

Comment:
`for relating_building_element, related_opening_elements in ...` - just to match IFC jargon